test_queue.py

```diff
@@ -12,20 +13,29 @@ def populated_queue(request):
     return q
 
 
+def test_Node(request):
+    assert Node(5).val == 5
+
+
+def test_Queue(request):
+    assert Queue().head == Queue().tail
+    assert Queue().size() == 0
+
+
 def test_enqueue(request):
     q = Queue()
     q.enqueue(5)
     assert q.head.val == q.tail.val
     assert q.head.val == 5
-    assert q.size == 1
+    assert q.size() == 1
 
 
 def test_dequeue(request, populated_queue):
     assert populated_queue.dequeue() == 5
     assert populated_queue.head.val == "True"
-    assert populated_queue.size == 3
+    assert populated_queue.size() == 3
 
 
 def test_size(request, populated_queue):
-    assert populated_queue.size == 4
-    assert Queue().size == 0
+    assert populated_queue.size() == 4
+    assert Queue().size() == 0
```

Comment:
You didn't test your error case, and you didn't exhaustively test your size() function. Think of ways your size function could break, and write tests to make sure they work properly